tools/wptrunner/OWNERS

```diff
@@ -0,0 +1,2 @@
+@bobholt
+@jgraham
```

Comment:
Ah. Couldn't remember if we settled on notifications being recursive or fallback. Updated.